griptape/tools/google_drive/tool.py

```diff
@@ -45,7 +45,7 @@ def list_files(self, params: dict) -> ListArtifact | ErrorArtifact:
         folder_path = values.get("folder_path", self.DEFAULT_FOLDER_PATH)
 
         try:
-            service = self._build_client(self.LIST_FILES_SCOPES)
+            service = self._build_client(self.LIST_FILES_SCOPES, "drive", "v3", self.owner_email)
```

Comment:
Let's move those into `SERVICE_NAME`, and `SERVICE_VERSION` since they are strings repeated in multiple places in this class.

Reply:
Done!